### PR TITLE
Update api

### PR DESCRIPTION
**Pull request by amarjeet045**
in the api folder, there are 4 endpoints
/activity
/events
/timer
/seswebhook

in the services folder, there are two files
officeStatus.js is the template for getting the result of office which is pending
officeQuery.js is the file where i have passed all the office with status pending in officeStatus function.

in the utils folder,
there are constant defined which can be used anywhere

### CODE REVIEW

File: functions/api/event.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const { code } = require('../utils/reportUtils');
+const { getBearerToken, tokenValidator } = require('../utils/http');
+
+const event = async (req, res) => {
+  try {
+    const bearerToken = getBearerToken({ headers: req.headers })
+    if (tokenValidator(bearerToken)) {
+      const eventCollection = await growthfilemsdb
+        .collection('Events')
+        .add(req.body, { merge: false });
```
Comment (utkarshbhatt12) [inline functions/api/event.js:11]: `{ merge: false }` isn't supported in `.add` function

File: functions/api/updateApi.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    const bearerToken = getBearerToken({ headers: req.headers })
+    if (tokenValidator(bearerToken)) {
+      const officeID = req.body.officeId;
+      const recipientID = req.body.recipientId;
+
+      const data = {
+        requestBody: req.body,
+      };
+      if (
+        req.body.hasOwnProperty('officeId') &&
```
Comment (utkarshbhatt12) [inline functions/api/updateApi.js:15]: What if `req.body.office` is an empty string?

File: functions/api/updateApi.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          .collection('Recipients')
+          .doc(recipientID)
+          .set(data.requestBody, { merge: false });
+
+        return res.status(code.ok).send({ success: true, message: `Success` });
+      }
+    } else {
+      res.status(code.forbidden).send('Forbidden');
+    }
+  } catch (error) {
```
Comment (utkarshbhatt12) [inline functions/api/updateApi.js:41]: Log the error in console.

Comment (amarjeet045) [reply functions/api/updateApi.js:41]: ok

**Pull request merged**